tests/unit/modules/network/nxos/test_nxos_interfaces.py

```diff
@@ -553,7 +564,9 @@ def test_3(self):
             "interface loopback1",
             "shutdown",
             "interface loopback4",
+            "no shutdown",
             "interface port-channel4",
+            "no shutdown",
             "interface Ethernet1/4.101",
```

Comment:
Based on the given playbook and the existing config, yes it should push `no shutdown`. However, looking at it more, it seems that the default state of new port-channels is enabled, since `shutdown` appears in running-config for them, but not `no shutdown`, unlike Ethernet or Vlan interfaces. I need to confirm if that's the case across all NX-OS versions.

@praveenramoorthy could you please help with this? Thanks.

Reply:
The default status of virtual interfaces on various NXOS versions are as follows.

N9k - (9.3.5 & 7.0.3)
=============
vlan            - shutdown
loopback        - no shutdown
port-channel    - no shutdown

N7K - (8.3.2 & 7.3.0)
==============
vlan            - shutdown
loopback        - no shutdown
port-channel    - no shutdown

N3k - (9.3.8)
=============
vlan            - shutdown
loopback        - no shutdown
port-channel    - no shutdown

N6K - (7.3.3)
=============
vlan            - shutdown
loopback        - no shutdown
port-channel    - no shutdown

Thanks.

Reply:
@NilashishC I was referring to the test config you have in there for sub-interfaces, not port-channels. I'm running this on a 9k on 9.3(7), FYI. Also, while "no shutdown" does not appear in the running-config for port-channels, it does appear for vlan interfaces and sub-interfaces.

Reply:
@zendritic Sorry for the really long delay on this one. I don't think there should be a `no shutdown` command for `interface Ethernet1/4.101` here. This sub-interface does not exist in the existing config and the playbook (for the states) just defines it without specifying `enabled: True/False`.